packages/twenty-front/src/modules/spreadsheet-import/utils/setColumn.ts

```diff
@@ -12,33 +13,56 @@ export const setColumn = <T extends string>(
   field?: Field<T>,
   data?: MatchColumnsStepProps<T>['data'],
 ): Column<T> => {
-  switch (field?.fieldType.type) {
```

Comment:
Why removing the switch case pattern here? If it's too big maybe we can split the code inside functions?

Reply:
Tbh I'm not sure why but the switch/case caused a wired typescript issue which I didn't figure out